edudoc/disciplinary-crash-course.md

```diff
@@ -0,0 +1,96 @@
+* Report incidents of theft to the WDC so that we can note down competitors names in case there are repeated cases of theft in the future.
+
+# Handling Behavioral Issues of Spectators at Competitions {.page-break-before}
+
+* Local Delegates should do their best to handle any issues that arise with spectators in a respectful manner, remembering that they are acting as official representatives of the WCA.
+* Be aware of venue rules and policies. You may be able to ask an official from the venue or local law enforcement to remove individuals causing problems if needed. 
+* Contact local law enforcement immediately if there is extreme behavior that may threaten competitor safety. Once the immediate concerns of safety have been addressed, contact the WDC to report the details of what happened.
+
+# Online Interactions (WCA Forum/Official WCA Platforms)
+
+* If competitors/parents/others are being impolite or inappropriate in their online interactions related to competitions, then please do your best to respectfully handle the situation (please keep in mind that you are a representative of the WCA). You may reach out to the WDC for advice on how to best handle the situation if the problem persists or you are unsure of how to proceed.
+* If you feel a person may engage in disruptive actions or pose a threat to the safety or well-being or anyone during a WCA-sanctioned competition, then please contact the WDC.
+* For any incidents on the WCA forum or other official WCA platforms please contact the moderators, such as the WAC on the WCA forums. The WDC would like for minor behavioral incidents to be handled “locally” online on official WCA platforms– such as through the means of warnings, mutes, or bans–before the WDC is contacted to request disciplinary action being enacted. Please only contact the WDC for serious incidents that take place online on official WCA platforms.
+
+# Organizers
+
+* Like any other competitors, report any violations of the Code of Conduct to the WDC.
+* If an organizer has done something outside of the scope of the Code of Conduct that you feel could be problematic, then it is up to discretion of the Delegate to work with that organizer or not. For example, if an organizer has made racist comments on a non-WCA online forum, then the Delegate can choose to not work with that organizer. 
+* Carefully keep track of competition budgets and actual expenditures that take place, documenting expenditures with receipts when possible. If you believe that an organizer or legal entity is retaining leftover money from competitions for personal gain then please contact the WDC.
+
+# WCA Staff Members {.page-break-before}
+
+* Like any other competitors, report any violations of the Code of Conduct of WCA Staff to the WDC. Be sure to also use ‘CC’ to include the WCA Ethics Committee (WEC) in your email.
```

Comment:
It is worth explicitly making the comparison to situations where one would report a staff member to WEC instead of WDC, even if it's technically out of scope of this document. Right now it seems to imply that all staff infractions should go to WDC, unless they are on WDC themselves.

Reply:
That is correct, if a staff member violates CoC then it should go to the WDC. The WDC will forward it to the WEC if it’s an ethical issue as well